### DISCUSSION TITLE
pnpm ^C command ELIFECYCLE Command Failed in WSL2

### DISCUSSION THREAD

**Discussion by ThimoDEV** [pnpm/pnpm#6069] [Q&A]
Does someone have an idea what could be the problem of this issue or what I could try to fix it? 

- I use wsl2 to run my nuxt 3 app with pnpm. 
- I enabled corepack
- Each time I close my dev environment with ctrl C this command failed pops up

It doesn't happen in windows, only in wsl2
![image](https://user-images.githubusercontent.com/68739517/218144802-ab727dd1-8cd7-4125-a89f-1c6b717b5f48.png)

**Comment by iobuhov** [comment]
Well, it looks like during execution pnpm/nuxt process revived SIGINT

> Each time I close my dev environment with ctrl C this command failed pops up

`ctrl+c` is not a keyboard short-cut from IDE, it's special terminal seq, which is a reason why your process is stopped
https://learn.microsoft.com/en-us/windows/console/ctrl-c-and-ctrl-break-signals

**Reply by ThimoDEV** [reply]
Do you have an idea how I can workaround it?

**Reply by iobuhov** [reply]
Don't press `ctrl+c`? 🤷
